### PR TITLE
Updated what's new entry for color comparision method

### PR DESCRIPTION
**Pull request by surana-mudit**
Issue #10069 

## PR Summary
I have updated the what's new entry for the color comparision method. I think we can close the old PR #10071 

## PR Checklist

- [x] Has Pytest style unit tests
- [x] Code is PEP 8 compliant
- [x] New features are documented, with examples if plot related
- [x] Documentation is sphinx and numpydoc compliant
- [x] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [x] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/users/next_whats_new/2017_12_17_compare_colors.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Comparision of 2 colors in Matplotlib
+-------------------------------------
+
+As the colors in Matplotlib can be specified with a wide variety of ways, this new version consists of a method ``matplotlib.colors.same_color`` which allows 2 legend colors to be compared.
```
Comment (jklymak) [inline doc/users/next_whats_new/2017_12_17_compare_colors.rst]: Just use single ticks so this generates a link.  

The English is a bit convoluted..:

```
New method `matplotlib.colors.same_color` checks if  two `~matplotlib.colors` are the same.  
```

**Comment by surana-mudit** [comment]
@jklymak  I am not able to find any way to generate link to the method. I have also tried your idea but its not working.

**Comment by dstansby** [comment]
Sphinx error seems unrelated, if someone merges this please can it be squashed.

**Comment by surana-mudit** [comment]
@dstansby  @jklymak  any updates on this PR?

**Pull request merged**